website/docs/r/glue_connection.html.markdown

```diff
@@ -83,7 +83,7 @@ resource "aws_glue_connection" "example_connector" {
     CONNECTOR_CLASS_NAME = "net.snowflake.client.jdbc.SnowflakeDriver"
     CONNECTION_TYPE      = "Jdbc"
     CONNECTOR_URL        = "s3://example/snowflake-jdbc.jar" # S3 path to the snowflake jdbc jar
-    JDBC_CONNECTION_URL  = "[[\"default=jdbc:snowflake://example.com/?user=$${user}&password=$${password}\"],\",\"]"
+    JDBC_CONNECTION_URL  = "[[\"default=jdbc:snowflake//example.com/?user=$${user}&password=$${password}\"],\",\"]"
```

Comment:
```suggestion
    JDBC_CONNECTION_URL  = "[[\"default=jdbc:snowflake:&#8203;//example.com/?user=$${user}&password=$${password}\"],\",\"]"
```

Try zero-width space trick?
* https://github.com/github/markup/issues/1153#issuecomment-450893185
* https://stackoverflow.com/questions/50657698/how-can-i-stop-github-from-rendering-emojis-in-markdown